### PR TITLE
Fix:  `fatal error: fault` panic

### PR DESCRIPTION
**Pull request by dongjiang1989**
Fix `apiserver` panic:
```
unexpected fault address 0x80808e47f7649
fatal error: fault
[signal SIGSEGV: segmentation violation code=0x1 addr=0x80808e47f7649 pc=0x7b764]

goroutine 1805 [running]:
runtime.throw({0x2f3d33c?, 0x400544a0e0?})
	/usr/local/go/src/runtime/panic.go:1077 +0x40 fp=0x400623cfa0 sp=0x400623cf70 pc=0x4a5e0
runtime.sigpanic()
	/usr/local/go/src/runtime/signal_unix.go:875 +0x22c fp=0x400623d000 sp=0x400623cfa0 pc=0x6191c
aeshashbody()
	/usr/local/go/src/runtime/asm_arm64.s:792 +0x2d4 fp=0x400623d010 sp=0x400623d010 pc=0x7b764
runtime.mapiternext(0x4005379b80)
	/usr/local/go/src/runtime/map.go:937 +0x2c4 fp=0x400623d080 sp=0x400623d010 pc=0x21634
runtime.mapiterinit(0xc00000000000030?, 0x4002bf8f28?, 0xc99e9d082d1ef96?)
	/usr/local/go/src/runtime/map.go:864 +0x2a0 fp=0x400623d0b0 sp=0x400623d080 pc=0x21330
reflect.mapiterinit(0x4002bf91e3?, 0x4002bf9280?, 0x4005379b80?)
	/usr/local/go/src/runtime/map.go:1392 +0x1c fp=0x400623d0e0 sp=0x400623d0b0 pc=0x779bc
github.com/modern-go/reflect2.(*UnsafeMapType).UnsafeIterate(...)
	/workspace/vendor/github.com/modern-go/reflect2/unsafe_map.go:112
github.com/json-iterator/go.(*mapEncoder).IsEmpty(0x400623d1e8?, 0x1edd8?)
	/workspace/vendor/github.com/json-iterator/go/reflect_map.go:275 +0x2c fp=0x400623d120 sp=0x400623d0e0 pc=0x4c554c
github.com/json-iterator/go.(*structFieldEncoder).IsEmpty(0x4005447a40, 0x29322a0?)
	/workspace/vendor/github.com/json-iterator/go/reflect_struct_encoder.go:118 +0x4c fp=0x400623d140 sp=0x400623d120 pc=0x4d189c
github.com/json-iterator/go.(*structEncoder).Encode(0x4005447a70, 0x4001888400?, 0x400090b440)
	/workspace/vendor/github.com/json-iterator/go/reflect_struct_encoder.go:148 +0x268 fp=0x400623d230 sp=0x400623d140 pc=0x4d1be8
github.com/json-iterator/go.(*OptionalEncoder).Encode(0x2975a60?, 0x400041caa0?, 0x0?)
	/workspace/vendor/github.com/json-iterator/go/reflect_optional.go:70 +0xac fp=0x400623d290 sp=0x400623d230 pc=0x4ca63c
github.com/json-iterator/go.(*onePtrEncoder).Encode(0x4005443260, 0x4000768260, 0x4005447800?)
	/workspace/vendor/github.com/json-iterator/go/reflect.go:219 +0x80 fp=0x400623d2d0 sp=0x400623d290 pc=0x4be440
github.com/json-iterator/go.(*Stream).WriteVal(0x400090b440, {0x2975a60, 0x4000768260})
	/workspace/vendor/github.com/json-iterator/go/reflect.go:98 +0x15c fp=0x400623d350 sp=0x400623d2d0 pc=0x4bd89c
github.com/json-iterator/go.(*dynamicEncoder).Encode(0xffff50768e60?, 0x5f1be10?, 0x400623d3c8?)
	/workspace/vendor/github.com/json-iterator/go/reflect_dynamic.go:15 +0x3c fp=0x400623d380 sp=0x400623d350 pc=0x4bfccc
github.com/json-iterator/go.(*sliceEncoder).Encode(0x4005436df8, 0x40051819a0, 0x400090b440)
	/workspace/vendor/github.com/json-iterator/go/reflect_slice.go:38 +0x2ac fp=0x400623d440 sp=0x400623d380 pc=0x4cb00c
github.com/json-iterator/go.(*placeholderEncoder).Encode(0x1?, 0x4005442f00?, 0x400623d4e8?)
	/workspace/vendor/github.com/json-iterator/go/reflect.go:332 +0x28 fp=0x400623d470 sp=0x400623d440 pc=0x4bedd8
github.com/json-iterator/go.(*structFieldEncoder).Encode(0x4005447620, 0x265adcd?, 0x400090b440)
	/workspace/vendor/github.com/json-iterator/go/reflect_struct_encoder.go:110 +0x54 fp=0x400623d4f0 sp=0x400623d470 pc=0x4d1714
github.com/json-iterator/go.(*structEncoder).Encode(0x4005447710, 0x4d3064?, 0x400090b440)
	/workspace/vendor/github.com/json-iterator/go/reflect_struct_encoder.go:158 +0x3f4 fp=0x400623d5e0 sp=0x400623d4f0 pc=0x4d1d74
github.com/json-iterator/go.(*OptionalEncoder).Encode(0x400623d601?, 0x40054430f0?, 0x400623d6b8?)
	/workspace/vendor/github.com/json-iterator/go/reflect_optional.go:70 +0xac fp=0x400623d640 sp=0x400623d5e0 pc=0x4ca63c
github.com/json-iterator/go.(*structFieldEncoder).Encode(0x4005447770, 0x26630b1?, 0x400090b440)
	/workspace/vendor/github.com/json-iterator/go/reflect_struct_encoder.go:110 +0x54 fp=0x400623d6c0 sp=0x400623d640 pc=0x4d1714
github.com/json-iterator/go.(*structEncoder).Encode(0x40054477a0, 0x4001888400?, 0x400090b440)
	/workspace/vendor/github.com/json-iterator/go/reflect_struct_encoder.go:158 +0x3f4 fp=0x400623d7b0 sp=0x400623d6c0 pc=0x4d1d74
github.com/json-iterator/go.(*OptionalEncoder).Encode(0x400623d848?, 0x4bd850?, 0x2a5ede0?)
	/workspace/vendor/github.com/json-iterator/go/reflect_optional.go:70 +0xac fp=0x400623d810 sp=0x400623d7b0 pc=0x4ca63c
github.com/json-iterator/go.(*onePtrEncoder).Encode(0x4005443110, 0x40007681d8, 0x400623d8a8?)
	/workspace/vendor/github.com/json-iterator/go/reflect.go:219 +0x80 fp=0x400623d850 sp=0x400623d810 pc=0x4be440
github.com/json-iterator/go.(*Stream).WriteVal(0x400090b440, {0x2c70260, 0x40007681d8})
	/workspace/vendor/github.com/json-iterator/go/reflect.go:98 +0x15c fp=0x400623d8d0 sp=0x400623d850 pc=0x4bd89c
github.com/json-iterator/go.(*dynamicEncoder).Encode(0x4005447440?, 0x4005447170?, 0x400623d948?)
	/workspace/vendor/github.com/json-iterator/go/reflect_dynamic.go:15 +0x3c fp=0x400623d900 sp=0x400623d8d0 pc=0x4bfccc
github.com/json-iterator/go.(*sliceEncoder).Encode(0x4005436df8, 0x4005181968, 0x400090b440)
	/workspace/vendor/github.com/json-iterator/go/reflect_slice.go:38 +0x2ac fp=0x400623d9c0 sp=0x400623d900 pc=0x4cb00c
github.com/json-iterator/go.(*placeholderEncoder).Encode(0x56b9401?, 0x4005442f00?, 0x400623da68?)
	/workspace/vendor/github.com/json-iterator/go/reflect.go:332 +0x28 fp=0x400623d9f0 sp=0x400623d9c0 pc=0x4bedd8
github.com/json-iterator/go.(*structFieldEncoder).Encode(0x4005447650, 0x265aded?, 0x400090b440)
	/workspace/vendor/github.com/json-iterator/go/reflect_struct_encoder.go:110 +0x54 fp=0x400623da70 sp=0x400623d9f0 pc=0x4d1714
github.com/json-iterator/go.(*structEncoder).Encode(0x4005447710, 0x4d3064?, 0x400090b440)
	/workspace/vendor/github.com/json-iterator/go/reflect_struct_encoder.go:158 +0x3f4 fp=0x400623db60 sp=0x400623da70 pc=0x4d1d74
github.com/json-iterator/go.(*OptionalEncoder).Encode(0x400623db01?, 0x40054430f0?, 0x400623dc38?)
	/workspace/vendor/github.com/json-iterator/go/reflect_optional.go:70 +0xac fp=0x400623dbc0 sp=0x400623db60 pc=0x4ca63c
github.com/json-iterator/go.(*structFieldEncoder).Encode(0x4005447770, 0x26630b1?, 0x400090b440)
	/workspace/vendor/github.com/json-iterator/go/reflect_struct_encoder.go:110 +0x54 fp=0x400623dc40 sp=0x400623dbc0 pc=0x4d1714
github.com/json-iterator/go.(*structEncoder).Encode(0x40054477a0, 0x4001888400?, 0x400090b440)
	/workspace/vendor/github.com/json-iterator/go/reflect_struct_encoder.go:158 +0x3f4 fp=0x400623dd30 sp=0x400623dc40 pc=0x4d1d74
github.com/json-iterator/go.(*OptionalEncoder).Encode(0x2c70260?, 0x400041caa0?, 0x0?)
	/workspace/vendor/github.com/json-iterator/go/reflect_optional.go:70 +0xac fp=0x400623dd90 sp=0x400623dd30 pc=0x4ca63c
github.com/json-iterator/go.(*onePtrEncoder).Encode(0x4005443110, 0x4000768100, 0x400542f1a0?)
	/workspace/vendor/github.com/json-iterator/go/reflect.go:219 +0x80 fp=0x400623ddd0 sp=0x400623dd90 pc=0x4be440
github.com/json-iterator/go.(*Stream).WriteVal(0x400090b440, {0x2c70260, 0x4000768100})
	/workspace/vendor/github.com/json-iterator/go/reflect.go:98 +0x15c fp=0x400623de50 sp=0x400623ddd0 pc=0x4bd89c
github.com/json-iterator/go.(*dynamicEncoder).Encode(0x3?, 0x4?, 0x400623deb8?)
	/workspace/vendor/github.com/json-iterator/go/reflect_dynamic.go:15 +0x3c fp=0x400623de80 sp=0x400623de50 pc=0x4bfccc
github.com/json-iterator/go.(*sliceEncoder).Encode(0x40054368e8, 0x4005181900, 0x400090b440)
	/workspace/vendor/github.com/json-iterator/go/reflect_slice.go:38 +0x2ac fp=0x400623df40 sp=0x400623de80 pc=0x4cb00c
github.com/json-iterator/go.(*placeholderEncoder).Encode(0x400623df01?, 0x40054423a0?, 0x400623dfe8?)
	/workspace/vendor/github.com/json-iterator/go/reflect.go:332 +0x28 fp=0x400623df70 sp=0x400623df40 pc=0x4bedd8
github.com/json-iterator/go.(*structFieldEncoder).Encode(0x400542f950, 0x265adcd?, 0x400090b440)
	/workspace/vendor/github.com/json-iterator/go/reflect_struct_encoder.go:110 +0x54 fp=0x400623dff0 sp=0x400623df70 pc=0x4d1714
github.com/json-iterator/go.(*structEncoder).Encode(0x400542fa40, 0x1bfd8?, 0x400090b440)
	/workspace/vendor/github.com/json-iterator/go/reflect_struct_encoder.go:158 +0x3f4 fp=0x400623e0e0 sp=0x400623dff0 pc=0x4d1d74
github.com/json-iterator/go.(*OptionalEncoder).Encode(0x400623e178?, 0xffff4fe597b0?, 0x400623e188?)
	/workspace/vendor/github.com/json-iterator/go/reflect_optional.go:70 +0xac fp=0x400623e140 sp=0x400623e0e0 pc=0x4ca63c
github.com/json-iterator/go.(*placeholderEncoder).Encode(0x400623e101?, 0x40054424c0?, 0x400623e1e8?)
	/workspace/vendor/github.com/json-iterator/go/reflect.go:332 +0x28 fp=0x400623e170 sp=0x400623e140 pc=0x4bedd8
github.com/json-iterator/go.(*structFieldEncoder).Encode(0x400542fbc0, 0x26630b1?, 0x400090b440)
	/workspace/vendor/github.com/json-iterator/go/reflect_struct_encoder.go:110 +0x54 fp=0x400623e1f0 sp=0x400623e170 pc=0x4d1714
github.com/json-iterator/go.(*structEncoder).Encode(0x400542fbf0, 0x1bfd8?, 0x400090b440)
	/workspace/vendor/github.com/json-iterator/go/reflect_struct_encoder.go:158 +0x3f4 fp=0x400623e2e0 sp=0x400623e1f0 pc=0x4d1d74
github.com/json-iterator/go.(*OptionalEncoder).Encode(0x400623e368?, 0xffff4fe59790?, 0x400623e388?)
	/workspace/vendor/github.com/json-iterator/go/reflect_optional.go:70 +0xac fp=0x400623e340 sp=0x400623e2e0 pc=0x4ca63c
github.com/json-iterator/go.(*structFieldEncoder).Encode(0x400542fc50, 0x4d3064?, 0x400090b440)
	/workspace/vendor/github.com/json-iterator/go/reflect_struct_encoder.go:110 +0x54 fp=0x400623e3c0 sp=0x400623e340 pc=0x4d1714
github.com/json-iterator/go.(*dereferenceEncoder).Encode(0x4c1c2dcea1966c01?, 0x4000769000?, 0x400623e498?)
	/workspace/vendor/github.com/json-iterator/go/reflect_optional.go:86 +0xac fp=0x400623e420 sp=0x400623e3c0 pc=0x4ca73c
github.com/json-iterator/go.(*structFieldEncoder).Encode(0x400542fc80, 0x26426af?, 0x400090b440)
	/workspace/vendor/github.com/json-iterator/go/reflect_struct_encoder.go:110 +0x54 fp=0x400623e4a0 sp=0x400623e420 pc=0x4d1714
github.com/json-iterator/go.(*structEncoder).Encode(0x40054470b0, 0x4001888400?, 0x400090b440)
	/workspace/vendor/github.com/json-iterator/go/reflect_struct_encoder.go:158 +0x3f4 fp=0x400623e590 sp=0x400623e4a0 pc=0x4d1d74
github.com/json-iterator/go.(*OptionalEncoder).Encode(0x2dea500?, 0x400041caa0?, 0x0?)
	/workspace/vendor/github.com/json-iterator/go/reflect_optional.go:70 +0xac fp=0x400623e5f0 sp=0x400623e590 pc=0x4ca63c
github.com/json-iterator/go.(*onePtrEncoder).Encode(0x4005442e00, 0x400519cd40, 0x400542ec30?)
	/workspace/vendor/github.com/json-iterator/go/reflect.go:219 +0x80 fp=0x400623e630 sp=0x400623e5f0 pc=0x4be440
github.com/json-iterator/go.(*Stream).WriteVal(0x400090b440, {0x2dea500, 0x400519cd40})
	/workspace/vendor/github.com/json-iterator/go/reflect.go:98 +0x15c fp=0x400623e6b0 sp=0x400623e630 pc=0x4bd89c
github.com/json-iterator/go.(*frozenConfig).Marshal(0x400041caa0, {0x2dea500, 0x400519cd40})
	/workspace/vendor/github.com/json-iterator/go/config.go:299 +0x88 fp=0x400623e750 sp=0x400623e6b0 pc=0x4b4fc8
github.com/json-iterator/go.Marshal(...)
	/workspace/vendor/github.com/json-iterator/go/adapter.go:34
kubesphere.io/kubesphere/pkg/simple/client/es/query.(*Builder).Bytes(0x4000435960?)
	/workspace/pkg/simple/client/es/query/builder.go:26 +0x40 fp=0x400623e780 sp=0x400623e750 pc=0x1fc58f0
kubesphere.io/kubesphere/pkg/simple/client/es.(*Client).Search(0x4000435960, 0xffff9793a5e0?, {0x400623e8b8?, 0x1f21c?, 0x54f8440?}, {0x18?, 0x18?, 0x54f8440?}, 0x98?)
	/workspace/pkg/simple/client/es/client.go:161 +0x58 fp=0x400623e830 sp=0x400623e780 pc=0x25706a8
kubesphere.io/kubesphere/pkg/simple/client/auditing/elasticsearch.(*client).CountOverTime(0x4000768520, 0x4001cfe380, {0x40034f5280, 0x3})
	/workspace/pkg/simple/client/auditing/elasticsearch/elasticsearch.go:68 +0x1e0 fp=0x400623e8f0 sp=0x400623e830 pc=0x2571190
kubesphere.io/kubesphere/pkg/models/auditing.(*eventsOperator).Events(0x400049fae0, 0x4005ea2300, 0x40051b3d70)
	/workspace/pkg/models/auditing/events.go:81 +0xee4 fp=0x400623eae0 sp=0x400623e8f0 pc=0x1f7cc44
kubesphere.io/kubesphere/pkg/models/tenant.(*tenantOperator).Auditing(0x4000e95ae0, {0x36d0938, 0x40041bc740}, 0x4005ea2300)
	/workspace/pkg/models/tenant/tenant.go:1113 +0xa34 fp=0x400623eee0 sp=0x400623eae0 pc=0x1f8eb74
kubesphere.io/kubesphere/pkg/kapis/tenant/v1alpha2.(*tenantHandler).Auditing(0x4000e115f0, 0x400280f560, 0x4005224fb8?)
	/workspace/pkg/kapis/tenant/v1alpha2/handler.go:404 +0x168 fp=0x400623ef70 sp=0x400623eee0 pc=0x1f925f8
kubesphere.io/kubesphere/pkg/kapis/tenant/v1alpha2.(*tenantHandler).Auditing-fm(0x196ce0e100000000?, 0x670cdd8d?)
	<autogenerated>:1 +0x34 fp=0x400623efa0 sp=0x400623ef70 pc=0x1fa3fc4
github.com/emicklei/go-restful.(*FilterChain).ProcessFilter(0x4000ea7c80?, 0x2fa5744?, 0x4005225008?)
	/workspace/vendor/github.com/emicklei/go-restful/filter.go:21 +0x74 fp=0x400623efd0 sp=0x400623efa0 pc=0x1070ed4
kubesphere.io/kubesphere/pkg/apiserver.monitorRequest(0x400280f560, 0x40014372d0, 0x40052250f8?)
	/workspace/pkg/apiserver/apiserver.go:197 +0x4c fp=0x400623f0d0 sp=0x400623efd0 pc=0x1fbea9c
github.com/emicklei/go-restful.(*FilterChain).ProcessFilter(0x4005225138?, 0x1e934?, 0x50?)
	/workspace/vendor/github.com/emicklei/go-restful/filter.go:19 +0x5c fp=0x400623f100 sp=0x400623f0d0 pc=0x1070ebc
kubesphere.io/kubesphere/pkg/apiserver.logRequestAndResponse(0x400280f560, 0x40014372d0, 0xffff50153401?)
	/workspace/pkg/apiserver/apiserver.go:617 +0x4c fp=0x400623f320 sp=0x400623f100 pc=0x1fc457c
github.com/emicklei/go-restful.(*FilterChain).ProcessFilter(0x4005ea2180?, 0x36bc5a8?, 0x400280f4d0?)
	/workspace/vendor/github.com/emicklei/go-restful/filter.go:19 +0x5c fp=0x400623f350 sp=0x400623f320 pc=0x1070ebc
github.com/emicklei/go-restful.(*Container).dispatch(0x40007b8000, {0x36bc5a8, 0x400280f4d0}, 0x40021a4f00)
	/workspace/vendor/github.com/emicklei/go-restful/container.go:291 +0x41c fp=0x400623f4a0 sp=0x400623f350 pc=0x106d24c
github.com/emicklei/go-restful.(*Container).dispatch-fm({0x36bc5a8?, 0x400280f4d0?}, 0x4005225518?)
	<autogenerated>:1 +0x40 fp=0x400623f4d0 sp=0x400623f4a0 pc=0x107aef0
net/http.HandlerFunc.ServeHTTP(0x4005225518?, {0x36bc5a8?, 0x400280f4d0?}, 0x1fc1fd0?)
	/usr/local/go/src/net/http/server.go:2136 +0x38 fp=0x400623f500 sp=0x400623f4d0 pc=0x373c48
net/http.(*ServeMux).ServeHTTP(0x4005225588?, {0x36bc5a8, 0x400280f4d0}, 0x40021a4f00)
	/usr/local/go/src/net/http/server.go:2514 +0x144 fp=0x400623f550 sp=0x400623f500 pc=0x375284
github.com/emicklei/go-restful.(*Container).ServeHTTP(0x4000d054a0?, {0x36bc5a8?, 0x400280f4d0?}, 0x8?)
	/workspace/vendor/github.com/emicklei/go-restful/container.go:311 +0x1bc fp=0x400623f5d0 sp=0x400623f550 pc=0x106db3c
kubesphere.io/kubesphere/pkg/apiserver/filters.WithKubeAPIServer.func1({0x36bc5a8, 0x400280f4d0}, 0x40021a4f00)
	/workspace/pkg/apiserver/filters/kubeapiserver.go:62 +0x37c fp=0x400623f730 sp=0x400623f5d0 pc=0x1d26d6c
net/http.HandlerFunc.ServeHTTP(0x36bc728?, {0x36bc5a8?, 0x400280f4d0?}, 0x400280f290?)
	/usr/local/go/src/net/http/server.go:2136 +0x38 fp=0x400623f760 sp=0x400623f730 pc=0x373c48
kubesphere.io/kubesphere/pkg/apiserver.(*APIServer).buildHandlerChain.WithAuditing.func1({0x36bc728, 0x4001f1d420}, 0x40052257f8?)
	/workspace/pkg/apiserver/filters/auditing.go:56 +0xe0 fp=0x400623f7d0 sp=0x400623f760 pc=0x1fc2010
net/http.HandlerFunc.ServeHTTP(0x36d0690?, {0x36bc728?, 0x4001f1d420?}, 0x1f21c?)
	/usr/local/go/src/net/http/server.go:2136 +0x38 fp=0x400623f800 sp=0x400623f7d0 pc=0x373c48
kubesphere.io/kubesphere/pkg/apiserver/filters.WithAuthorization.func1({0x36bc728, 0x4001f1d420}, 0x40021a4f00)
	/workspace/pkg/apiserver/filters/authorization.go:52 +0x254 fp=0x400623f8e0 sp=0x400623f800 pc=0x1d26154
net/http.HandlerFunc.ServeHTTP(0x36d0690?, {0x36bc728?, 0x4001f1d420?}, 0x3691918?)
	/usr/local/go/src/net/http/server.go:2136 +0x38 fp=0x400623f910 sp=0x400623f8e0 pc=0x373c48
kubesphere.io/kubesphere/pkg/apiserver/filters.WithAuthentication.func1({0x36bc728, 0x4001f1d420}, 0x40021a4f00)
	/workspace/pkg/apiserver/filters/authentication.go:70 +0x1f8 fp=0x400623fa40 sp=0x400623f910 pc=0x1d25a08
net/http.HandlerFunc.ServeHTTP(0x40021a4c00?, {0x36bc728?, 0x4001f1d420?}, 0x6?)
	/usr/local/go/src/net/http/server.go:2136 +0x38 fp=0x400623fa70 sp=0x400623fa40 pc=0x373c48
kubesphere.io/kubesphere/pkg/apiserver.(*APIServer).buildHandlerChain.WithRequestInfo.func2({0x36bc728, 0x4001f1d420}, 0x40021a4c00)
	/workspace/pkg/apiserver/filters/requestinfo.go:69 +0x27c fp=0x400623fb00 sp=0x400623fa70 pc=0x1fc1efc
net/http.HandlerFunc.ServeHTTP(0x56b9460?, {0x36bc728?, 0x4001f1d420?}, 0x4005225b40?)
	/usr/local/go/src/net/http/server.go:2136 +0x38 fp=0x400623fb30 sp=0x400623fb00 pc=0x373c48
net/http.serverHandler.ServeHTTP({0x4002e93110?}, {0x36bc728?, 0x4001f1d420?}, 0x6?)
	/usr/local/go/src/net/http/server.go:2938 +0xbc fp=0x400623fb60 sp=0x400623fb30 pc=0x37663c
net/http.(*conn).serve(0x400203ea20, {0x36d0690, 0x40035c9200})
	/usr/local/go/src/net/http/server.go:2009 +0x518 fp=0x400623ffa0 sp=0x400623fb60 pc=0x372a38
net/http.(*Server).Serve.func3()
	/usr/local/go/src/net/http/server.go:3086 +0x30 fp=0x400623ffd0 sp=0x400623ffa0 pc=0x376d50
runtime.goexit()
	/usr/local/go/src/runtime/asm_arm64.s:1197 +0x4 fp=0x400623ffd0 sp=0x400623ffd0 pc=0x7e564
created by net/http.(*Server).Serve in goroutine 1
	/usr/local/go/src/net/http/server.go:3086 +0x4cc
```

### What type of PR is this?

/kind bug

### Which issue(s) this PR fixes:

Fixes #

### Special notes for reviewers:
```
```

### Does this PR introduced a user-facing change?

```release-note
NONE
```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by wansir** [comment]
@dongjiang1989  Thank you for your contribution.

You can use `hack/pin-dependency.sh` to lock the dependency versions, and use `hack/update-vendor.sh` to update the vendor directory. This script will create symbolic links from `staging/src/kubesphere.io`.

**Comment by dongjiang1989** [comment]
> @dongjiang1989 Thank you for your contribution.
> 
> You can use `hack/pin-dependency.sh` to lock the dependency versions, and use `hack/update-vendor.sh` to update the vendor directory. This script will create symbolic links from `staging/src/kubesphere.io`.

Thanks @wansir. PTAL

File: vendor/kubesphere.io/api
Comment (wansir) [inline vendor/kubesphere.io/api:1]: Did you add all file changes after running update-vendor.sh? This symbolic link should not be deleted.

File: vendor/kubesphere.io/client-go
Comment (wansir) [inline vendor/kubesphere.io/client-go:1]: same here

Comment (dongjiang1989) [reply vendor/kubesphere.io/client-go:1]: Thanks. 
Done it.

File: vendor/kubesphere.io/api
Comment (dongjiang1989) [reply vendor/kubesphere.io/api:1]: Hmmm.... It's my bad. 
Fixed it done.
Thanks. @wansir Please re-check it.

**Pull request merged**